### DISCUSSION TITLE
@types/node 22.5.2 : In angular 14.3, which is using typescript 4.7.4 when I freshly install packages I get error related to circular dependency on build

### DISCUSSION THREAD

**Discussion by bhav-interspark** [DefinitelyTyped/DefinitelyTyped#70433] [Issues with a @types package]
angular@v14.3
typescript@v4.7.4
node@v14.21.3

Does @types/node 22.5.2 support typescript 4.7.4?

![image](https://github.com/user-attachments/assets/a2dd24d4-00d4-4767-8c32-77f4ec017e8c)

![image](https://github.com/user-attachments/assets/aa20c799-8c17-4d8c-b78e-356752a94019)

[a comment by an automated account is not shown]

**Comment by ThomasNB** [comment]
I think this a duplicate of https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/70432

**Reply by Renegade334** [reply]
In addition:

> Does @types/node 22.5.2 support typescript 4.7.4?

DT packages only support TypeScript versions from the previous two years; see the [support matrix](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#support-window).
